types/forge-apis/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Looks like our lint rules are disabled for this package. Can you clear our the disabled `rules` field and update your package appropriately?